webpack.config.prod.js

```diff
@@ -10,11 +10,11 @@ module.exports = {
   devtool: 'source-map',
 
   entry: {
-    app: ['./client/index.js'],
+    app: [ './client/index.js' ],
   },
 
   output: {
-    path: path.join(__dirname, 'public'),
+    path: path.join(__dirname, 'build'),
```

Comment:
The main idea to use 'build' is create-react-app. Nowadays we're getting a lot of newcomers to react community. They all goes through this starter kit. It would be ease for them to see similar naming.